examples/finner_grids/spe11a.txt

```diff
@@ -43,7 +43,7 @@ PERM6 10132499.7 PORO6 0.46
 PERM7          0 PORO7    0
 
 """Wells position"""
-"""radius, x, y, and z position [m] (final positions as well for spe11c)"""
+"""radius (0 to use sources instead, only available in Flow master), x, y, and z position [m] (final positions as well for spe11c)"""
```

Comment:
Suggest a bit more precise:
`(0 to use the SOURCE keyword instead of well keywords, this requires a Flow version newer than 23-01-2024)`

Reply:
Thanks, I have updated the text as you suggested.